experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -44,23 +46,28 @@ class Flowables {
   }
 
   template <typename T>
-  static Reference<Flowable<T>> just(std::initializer_list<T> list) {
-    auto lambda = [ list, it = list.begin() ](
+  static Reference<Flowable<T>> justN(std::initializer_list<T> list) {
+    std::vector<T> vec;
```

Comment:
Thanks for the link.  Yeah, that's unfortunate, but it makes sense though.  initializer_list<T> doesn't own the `T` values, so you shouldn't be able to move from them (and most users of it will make copies).

I'll switch it to using the copying ctor in vector.  I also considered having justN() be a variadic template, but it gets weird because the values in a parameter pack aren't required to be all the same type.